docs/index.md

```diff
@@ -53,10 +83,19 @@ This project was initially inspired by a [gist](https://gist.github.com/mosheesh
 
 ## Contributing
 
-See [CONTRIBUTING](https://github.com/testcontainers/testcontainers-java/blob/master/CONTRIBUTING.md).
+* Star the project on [Github](https://github.com/testcontainers/testcontainers-java) and help spread the word :)
+* Join our Slack: http://slack.testcontainers.org
```

Comment:
the Slack link is not clickable in the rendered result

Reply:
Thanks, will fix.